meal-planner/src/components/RecipePage.js

```diff
@@ -26,13 +35,18 @@ export default function RecipePage({
 
 const SectionStyled = styled.section`
     background: #800000fe;
```

Comment:
No variables in your globalstyle? Not judging... just observing... 😂

Reply:
:-)